src/Dialect/ONNX/ElementsAttr/Strides.hpp.inc

```diff
@@ -16,7 +16,7 @@ Iterator traverseStrides(llvm::ArrayRef<int64_t> shape, Iterator begin,
   auto traverse = [=](const auto &recurse, unsigned axis, Iterator it,
                       const Arg0 *elm0) -> Iterator {
     if (axis == shape.size()) {
-      act(it, *elm0);
+      act(it, elm0);
```

Comment:
the change makes traverseStrides(shape, begin, src, act) pass a const pointer to a value instead of passing a value by-value

you can see in mapStrides(shape, dst, src, act) that it now calls
```
traverseStrides(shape, dst.begin(), src, [&act](Res *res, const Arg0 *arg0) { *res = act(*arg0); })
```
instead of
```
traverseStrides(shape, dst.begin(), src, [&act](Res *res, Arg0 arg0) { *res = act(arg0); })
```

this change was useful because it allows ElementsAttrBuilder::reduce() to apply an offset when it reads the value pointer:
```
traverseStrides([&reducer, offset](WideNum *dst, const WideNum *src) { *dst = reducer(*dst, *(src + offset)); })
```

Reply:
I mixed const T * and T * const.